src/DefaultRenderer.js

```diff
@@ -211,6 +211,11 @@ export default class DefaultRenderer extends Component {
     }
 
     if (SceneComponent) {
+      if (navigationState.tabs || !navigationState.parent) {
```

Comment:
@phproot no problem! you can probably use getSceneStyle to account for the hideNavBar/hideTabBar props like in the Example app, and not hard-code a marginTop in sceneStyle.  Check the Example app that does exactly this.

That 3D screenshot is great.  Is that Reveal.app or something else?